### ISSUE TITLE
design: forbid negative line-height

### ISSUE THREAD

**Issue by crotru** [Decathlon/vitamin-web#30]
**Subject**

Negative line-height should be forbidden.
(WCAG recommends at least 1.5)

Right now we have this for large titles :

![image](https://user-images.githubusercontent.com/22882255/103235257-e8a8a380-4941-11eb-8bd3-2bc56a5acf3c.png)

This will break readability on multilines titles 
See: https://codepen.io/__sch__/pen/QWKaMmX

**Comment by lauthieb** [comment]
Thanks @crotru for reporting this issue.
FYI, we follow this guideline: https://www.decathlon.design/726f8c765/p/860e14-typography/b/208916
But yes, you're right, it's forbidden.
I report your issue to our designers!
Thanks :)

**Comment by MARIEDELATTRE** [comment]
Hi, Charles !
Thanks for your best eye :)
Normally for display 1 et 2 we do not recommend long sentences (see do/don't attached).
<img width="869" alt="Capture d’écran 2020-12-29 à 09 38 19" src="https://user-images.githubusercontent.com/61324309/103271262-04e82700-49ba-11eb-8b67-e54672ba39e7.png">
But I agree we should respect 1.5 regarding WCAG rules. We need to test if it will not be too spaced out (at this sizes 1.5 can be really fat).
I've create a ticket for that here :
[https://decathlon.atlassian.net/browse/DSMB-34](url)

**Comment by crotru** [comment]
Hey Marie 👋 
Sure 1.5 will be too big (this reco is more targeted to labor text anyway), 
but a value above 1 would be great 🙂 

About long sentences : we have titles with long words (yes I'm looking at you german fellows 🇩🇪 👀 ) that will span on 2 lines

**Comment by MARIEDELATTRE** [comment]
I had feedbacks from core team on display 2 1.5 is too much visually. We decide to have 1.2. Is it possible ? Because we'll have 57.6 on tablet and 38.4 on mobile.

**Comment by crotru** [comment]
1.2 is perfect 🙂 
Thanks Marie.

**Comment by crotru** [comment]
Actually even 1.1 would do for title line-spacing. 
The point is really to have something above 1.

**Comment by MARIEDELATTRE** [comment]
Ok thanks I will update decathlon.design with 1.2 on display 2 (we keep 1 on display 1)

**Comment by Sabrinavigil** [comment]
Hi @crotru, thanks for your feedback, thank to it we improved texts and displays :) 
To sum up : For Text 1, we changed line height from 26 to 27
For Text 3, from 22 to 21
For, Display 1 we moved to a 1 ratio and around a 1.1 ratio for Display 2.
Check just here our modifications, what do you think about ?   
<img width="406" alt="Ratio font" src="https://user-images.githubusercontent.com/65814825/104214038-44d9f000-5437-11eb-938b-ed1d82bf3f55.png">

**Comment by crotru** [comment]
Cool for me :)

**Comment by lauthieb** [comment]
Update, here are the new values available validated there: https://www.decathlon.design/726f8c765/p/860e14-typography/b/208916 ![image](https://user-images.githubusercontent.com/9600228/104650120-043bda00-56b6-11eb-886e-0c3387e1a317.png)
